cli/gluster-block.c

```diff
@@ -350,6 +350,20 @@ glusterBlockParseVolumeBlock(char *volumeblock, char *volume, char *block,
   return ret;
 }
 
+void getCommandString(char **cmd, int argcount, char **options)
+{
+   int total_length = 0;
+   int i;
+   for(i = 1; i< argcount; i++){
+	total_length = total_length + strlen(options[i])+1;
+   }
+   *cmd = (char *)malloc((total_length+1)*sizeof(char));
+   for(i=1; i< argcount; i++){
+	strcat(*cmd, options[i]);
+	strcat(*cmd, " ");
+   }
```

Comment:
should we add '\0' at the end of cmd after line 364 ?

Reply:
strcat(*cmd, " ") puts it automatically, so we don't need to put a '\0' explicitly after the loop